test/unit/windowHandlers.test.js

```diff
@@ -59,6 +59,17 @@ describe("Window handlers test", function () {
                 }
             }
         });
+
+        it("Throws correct error when calling recipe methods if SuperTokens is not initialized", async function () {
+            try {
+                await ThirdPartyPasswordless.getThirdPartyAuthorisationURLWithQueryParamsAndSetState({
+                    thirdPartyId: "google",
+                    frontendRedirectURI: "http://localhost:3000/auth/callback/google",
+                });
+            } catch (err) {
+                assert(err.message.startsWith("SuperTokens must be initialized before calling this method"));
+            }
+        });
```

Comment:
add tests for all recipes this way. You also need to add a version of the test in which you call supertokens.init, but not the recipe init and then you try and use the recipe function. The test here would fail saying that that recipe is not initialised (As opposed to supertokens not being initialised)

Reply:
Added tests for all recipes and also added corresponding tests where SuperTokens is initialized but the recipe is not